SQL/0000-00-00-schema.sql

```diff
@@ -980,6 +984,8 @@ CREATE TABLE `mri_violations_log` (
   PRIMARY KEY (`LogID`),
   CONSTRAINT `FK_tarchive_mriViolationsLog_1`
     FOREIGN KEY (`TarchiveID`) REFERENCES `tarchive` (`TarchiveID`),
+  CONSTRAINT `FK_mri_violations_log_scan_type`
+    FOREIGN KEY (`MriScanTypeID`) REFERENCES `mri_scan_type` (`ID`),
```

Comment:
fascinating, another foreign key I was sure was already there. 🤣 Aaah, the things you take for granted...